pkg/vfs/writer.go

```diff
@@ -294,14 +294,17 @@ func (f *fileWriter) Write(ctx meta.Context, off uint64, data []byte) syscall.Er
 		if f.totalSlices() < 1000 {
 			break
 		}
+		logger.Warnf("Too many pending slices of inode %d, pendings: %d, will wait for 1ms", f.inode, f.totalSlices())
```

Comment:
there will be too much logging

Reply:
OK, removed both logging. My point was - we need to tell user a write stall is happening now.